sdk/src/types/block/output/metadata.rs

```diff
@@ -141,14 +103,8 @@ pub mod dto {
         pub output_index: u16,
         pub is_spent: bool,
         #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
-        pub milestone_index_spent: Option<u32>,
-        #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
-        pub milestone_timestamp_spent: Option<u32>,
-        #[cfg_attr(feature = "serde", serde(skip_serializing_if = "Option::is_none"))]
-        pub transaction_id_spent: Option<String>,
-        pub milestone_index_booked: u32,
-        pub milestone_timestamp_booked: u32,
-        pub ledger_index: u32,
```

Comment:
Missing commitment_id_spent, transaction_id_spent?

Reply:
What do you mean? These are not part of the metadata

Reply:
What's this then? https://github.com/iotaledger/iota-core/blob/ddbddf41e8c222538e1e926f4e24d511bccfa477/documentation/APIs/openapi3-core.yaml#L2791-L2794

Reply:
Done